server/src/services/cleanUserInput.js

```diff
@@ -14,6 +14,16 @@ const cleanUserInput = formInput => {
         formInput.showId = validId
     }
 
+    if(field === "userId") {
+        const validId = Number(formInput.showId)
+        formInput.showId = validId
+    }
+
+    if(field === "reviewId") {
+        const validId = Number(formInput.showId)
+        formInput.showId = validId
+    }
```

Comment:
looks like these `if` statements are performing the same action, such that we could combine them:
```
if (field === "useId" || field === "reviewId") 
  ...
```
you can also add to the statement the other formats you're checking for Number (showId). You can then dynamically check with `Number(formInput[field])` instead of dot notation so it functions for all keys regardless of property name


Edit:
looks like these new if statements should instead be checking `formInput.reviewId`, etc, not `showId`

COPY AND PASTE DOES MORE HARM THAN GOOD!